src/get-started/java-binding.md

```diff
@@ -712,6 +712,49 @@ service_name1 = "/path/to/service1_artifact.jar"
 service_name2 = "/path/to/service2_artifact.jar"
 ```
 
+## Logging Configuration
+
+Java Binding uses two different methods for logging - [`log4j`][log4j-home]
+in Java modules and [`env_logger`][env_logger-home] in Rust modules.
+
+### Rust Logging
+
+Rust logs are produced by Exonum Core and can be used to monitor the
+status of the blockchain node, including information about the block
+commitment and the consensus status.
+
+Rust logs are disabled by default and controlled by the `RUST_LOG`
+environment variable. It is recommended to set `info` logging level
+for Exonum modules and `warn` level for all other modules:
+
+```bash
+export RUST_LOG=warn,exonum=info,exonum-java=info
+```
+
+Log entries go to `stderr` by default.
+
+See [`env_logger` documentation][env_logger-docs] for more information
+on possible configuration options.
+
+### Java Logging
+
+Java logs are produced by Java Binding internal code and can also be
```

Comment:
?

Logs produced by Java code (the framework and its dependencies, and the deployed services) are handled by Log4J framework. The services can use either Log4J or SLF4J logging APIs.